service/controller/resource/remotewritesecret/resource.go

```diff
@@ -0,0 +1,87 @@
+package remotewritesecret
+
+import (
+	"context"
+
+	"github.com/giantswarm/k8sclient/v7/pkg/k8sclient"
+	"github.com/giantswarm/micrologger"
+	promclient "github.com/prometheus-operator/prometheus-operator/pkg/client/versioned"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	pmov1alpha1 "github.com/giantswarm/prometheus-meta-operator/api/v1alpha1"
+	"github.com/giantswarm/prometheus-meta-operator/pkg/remotewriteutils"
+)
+
+const (
+	Name  = "remotewrite"
+	label = "monitoring.giantswarm.io/remotewrite"
```

Comment:
Using `monitoring.giantswarm.io` feels wrong, how about :

 ```suggestion
	label = "giantswarm.io/resource"
```

Reply:
I guess, I need to add another label that contains the name of `RemoteWrite` CR, otherwise we end up deleting other remote write secrets.